api/views.py

```diff
@@ -79,16 +84,31 @@ def post(self, request, *args, **kwargs):
         return self.create(request, *args, **kwargs)
 
 
-class SalaryListAPIView(mixins.CreateModelMixin, generics.ListAPIView):
-    serializer_class = SalaryListSerializer
-    pagination_class = LimitOffsetPagination
-
-    def get_queryset(self):
-        qs = Salary.objects.filter(company_employee__employee__id=self.kwargs['id'])
-        return qs
-
-    def get(self, request, *args, **kwargs):
-        return self.list(request, *args, **kwargs)
-
-    def post(self, request, *args, **kwargs):
-        return self.create(request, *args, **kwargs)
+class SalaryListAPIView(APIView):
+    def post(self, request):
+        company_employee_id, salary, date = request.POST.get('id'), request.POST.get('salary'), \
+                                            request.POST.get('date')
+        try:
+            salary_serializer = Validate.validate_by_serializer(SalaryParamsSerializer, SalaryParamsError,
+                                                                data={'company_employee_id': company_employee_id,
```

Comment:
`data=` should be identated properly to the left. 
Again, feel free to use multiple lines for better code look: 

```python
data = {
   'company': ...,
   'salary': ...
}
```